packages/offix-datastore/src/storage/adapters/IndexedDBStorage.ts

```diff
@@ -1,11 +1,11 @@
-import { Model, PersistedModel } from "../models";
-import { Storage } from "./Storage";
-import { getStoreNameFromModelName, generateId } from "./core";
-import { PredicateFunction } from "../predicates";
+import { Model, PersistedModel } from "../../models";
+import { IStorageAdapter } from "../Storage";
+import { getStoreNameFromModelName } from "../core";
+import { PredicateFunction } from "../../predicates";
 
 const DB_NAME = "offix-datastore";
 
-export class IndexedDBStorage implements Storage {
+export class IndexedDBStorage implements IStorageAdapter {
```

Comment:
Very cool, this makes it easier to add in async storage for React Native at a later stage.